esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -3488,10 +3504,17 @@ bool CWsDfuEx::doLogicalFileSearch(IEspContext &context, IUserDescriptor* udesc,
     ForEach(*it)
         addToLogicalFileList(it->query(), NULL, version, logicalFiles);
 
-    if (version >= 1.24)
-        resp.setCacheHint(cacheHint);
+    if ((oldCacheHint != cacheHint) && (totalFiles > maxFiles))
```

Comment:
Still unclear..
This is saying if oldCacheHit matches AND tottalFiles > maxFiles.. then *don't* issue warning afaics , which doesn't make a lot of sense to me..

Which I think will mean I will see the warning on the 1st page.. but if I immediately page to 2nd (therefore same hint), I will not see the warning... (?)